requirements.txt

```diff
@@ -1,4 +1,5 @@
 click
 Flask
+flask_cors
```

Comment:
Why is flask_cors now a required dependency?

Reply:
Think this is very useful for many client-apps, though in practice CORS would be handled by a front-end proxy like Apache, nginx, Traefik. NB shouldn't the name be `flask-cors`? (no underscore). It is already implemented [see here](https://github.com/geopython/pygeoapi/blob/master/pygeoapi/flask_app.py#L52).

Reply:
This complicates the debian install which now needs to provide python-flask-cors (or python3-flask-cors?) given `setup.py` is tied to `requirements.txt`.

The implementation in https://github.com/geopython/pygeoapi/blob/master/pygeoapi/flask_app.py#L52 is conditional, so if a user specifies cors in pygeoapi config, and the install doesn't have it, then it will fail on startup and a simple `pip install flask-cors` will do the trick.  I think this is a cleaner and safer approach than promoting flask-cors to the core of the implementation requirements.

Reply:
Ok, no strong opinion, this is typically something to document, also that the Docker image supports it (via `requirements-dev.txt`).

Reply:
Some clarification, from the back of my head (will not put my hands on fire for it) we had the `PYGEOAPI_CONFIG="pwd/pygeoapi-config.yml` with `cors:true` and I think that was the case when I was initially working with RTD. 

And this caused a problem when loading pygeoapi on the RTD platform. Now I see that the coors has been deactivated on the configuration file. So I will removed it from the dependencies

Reply:
Put Flask_cors back to requirements.txt as it was before

Reply:
Should we keep flask_cors in `requirements-dev.txt`?